src/core_plugins/elasticsearch/lib/expose_client.js

```diff
@@ -55,9 +55,19 @@ module.exports = function (server) {
       ssl.ca = options.ca.map(readFile);
     }
 
+    const host = {
+      host: uri.hostname,
+      port: uri.port,
+      protocol: uri.protocol.replace(':', ''),
```

Comment:
Yeah, https://github.com/elastic/elasticsearch-js/blob/master/test/unit/specs/host.js#L118-L119. Docs should be updated with that fact